### ISSUE TITLE
[false positive] windows update domains

### ISSUE THREAD

**Issue by dnmTX** [anudeepND/blacklist#30]
@anudeepND  currently you have three domains listed that would interfere with windows update:
`statsfe1.ws.microsoft.com`
`statsfe2.update.microsoft.com.akadns.net`
`statsfe2.ws.microsoft.com`

This is based out of [@crazy-max's lists](https://github.com/crazy-max/WindowsSpyBlocker/blob/master/data/hosts/update.txt) which is very accurate and curated frequently . Please check if there are more just in case as i found them by chance and i'm Windows user and need Windows Update working.
Thank you.

**Comment by anudeepND** [comment]
@dnmTX hmmm🤔 Windows 10 updates are working fine for me. I updated my laptop to 1809 build, if I remember correctly pi-hole logs didn't had these domains. Anyways I will look into it

**Comment by dnmTX** [comment]
@anudeepND  windows update is working here too,just sometimes it's like getting stuck on download for 20 or so min. and the resumes.My best guess is that it's trying to fetch the updates from blocked by me domain but there is no way to confirm my theory at the present.
Those listed above domains were moved at one point by @crazy-max from his tracking/telemetry lists to the update lists.I'm guessing there was a reason for that.
Anyway,thanks for looking in to it 👍

**Comment by dnmTX** [comment]
Found this as a reference,at least kind of explains what is the purpose of those domains:
https://www.wilderssecurity.com/threads/windows-update-statistics.362384/

**Comment by anudeepND** [comment]
What do you think @dnmTX? Remove them or leave them as it is?

**Comment by dnmTX** [comment]
Let's leave them for now.I changed my router and the firmware is kind of buggy,waiting on update from the developers.Once i got it stable i'll be able to monitor what's coming in and out.So far all that is just theory as i couldn't even find any official documentation what all those domains do. If i have proof we'll remove them.No rush on this one,as i said already windows update is working.
Closing this for now,will reopen if need to.
